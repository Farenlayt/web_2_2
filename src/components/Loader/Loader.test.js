import renderer from "react-test-renderer";
import {Provider} from "react-redux";
import storage from "../../store/FakeStorage";
import Loader from "../Loader/Loader";
import React from "react";

it('Main city loader is ok.', function() {
    const dom = renderer.create(
        <Provider store={storage}>
            <Loader isMajor={true}/>
        </Provider>
    ).toJSON();
    expect(dom).toMatchSnapshot();
});

it('Favour city loader is ok.', function() {
    const dom = renderer.create(
        <Provider store={storage}>
            <Loader isMajor={false}/>
        </Provider>
    ).toJSON();
    expect(dom).toMatchSnapshot();
});
