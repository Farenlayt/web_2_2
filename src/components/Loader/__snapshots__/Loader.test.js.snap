// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Favour city loader is ok. 1`] = `
<div
  className="favour-loader"
>
  Подождите, данные загружаются
</div>
`;

exports[`Main city loader is ok. 1`] = `
<div
  className="major-loader"
>
  Подождите, данные загружаются
</div>
`;
